dockernet/tests/integration_tests.bats

```diff
@@ -303,8 +281,72 @@ setup_file() {
   assert_equal "$sttoken_balance_diff" "$PACKET_FORWARD_STAKE_AMOUNT"
 }
 
+@test "[INTEGRATION-BASIC-$CHAIN_NAME] autopilot liquid stake and transfer" {
```

Comment:
Do you think we need to integration test redeeming twice from the same account in the same day epoch?

Reply:
imho, I think this type of test case granularity is best suited in a unit test (more specifically the redeem stake unit test).

That said, we do run two redeem stakes in a row in these IG tests (1 from autopilot and 1 as a regular redeem), so it probably doesn't hurt to tweak it a bit to make sure they occur in the same epoch!

Reply:
done [[here](https://github.com/Stride-Labs/stride/pull/1012/commits/f63a6a91c1ef850f6c63bdbfb928e7e52ccdc2c5)]